### ISSUE TITLE
Response in example and not examples

### ISSUE THREAD

**Issue by madhurabhogate** [outofcoffee/imposter#29]
I have a spec file in which the **example** is provided in the definition.
Just because there is no **examples** it doesn't send a response.

Here's an example:
```
---
swagger: "2.0"
info:
  version: "1.0.0"
  title: "Swagger Petstore"
consumes:
- "application/json"
produces:
- "application/json"
paths:
  /pets:
    get:
      description: "Returns all pets from the system"
      produces:
        - "application/json"
      responses:
        "200":
          description: "A list of pets."
          schema:
            items:
              $ref: "#/definitions/Pet"
definitions:
  Pet:
    type: "object"
    required:
      - "id"
      - "name"
    properties:
      id:
        type: "integer"
        format: "int64"
      name:
        type: "string"
    example:
      "id": 101,
      "name": "Cat"
```

**Comment by outofcoffee** [comment]
Hi @madhurabhogate, this is probably the same issue as #12, as currently examples are taken only from the 'responses' section not from models.

If you'd be interested in helping with #12 that'd be great.

**Comment by outofcoffee** [comment]
Hi @madhurabhogate, this was released in v1.2.0 (https://github.com/outofcoffee/imposter/pull/40).
